### DISCUSSION TITLE
Notion数据库数据收集

### DISCUSSION THREAD

**Discussion by paulprogrammer2001** [WeNeedHome/SummaryOfLoanSuspension#404] [General]
# 停贷数据规模统计 Notion数据库 

所有数据均来源于网络，数据真实性自行甄别。

**👏欢迎参与数据更新，友好合作，慎重更新。**
加入我们：https://stormy-thunbergia-586.notion.site/21dab14200e2478eb91c49b68d16495f

🚩请前往[](https://www.notion.so/936a4fc64e10486db84ee70426d150fe) 页添加数据

### 守则

- 禁止添加任何无关项
- 添加的所有停贷申明必须附上相关证明材料

### 概览

- 未提供业主数量的楼盘声明未纳入总业主数据统计数据。
- 未提供款项规模的楼盘声明未纳入预估总贷款规模数据。
- 由于无法证实数据来源的真实性和可靠性，总体情况仅供参考。

[[总体情况](https://www.notion.so/372043eeea6c4781895c7c5547cf6ac5)](https://www.notion.so/372043eeea6c4781895c7c5547cf6ac5)

## 相关数据详细页

[[SummaryOfLoanSuspension（可编辑）](https://www.notion.so/936a4fc64e10486db84ee70426d150fe)](https://www.notion.so/936a4fc64e10486db84ee70426d150fe)

**Comment by Jiyr0119** [comment]
给你点赞～

**Reply by paulprogrammer2001** [reply]
谢谢😘
